### ISSUE TITLE
Client unable to retrieve contract address from watcher's endpoints

### ISSUE THREAD

**Issue by Pongch** [omisego/elixir-omg#693]
**problem**
A user making transaction on the network needs to be able to retrieve the rootchain contract address in order to correctly sign a transaction on the childchain (following EIP712 standard). Right now this information is not exposed anywhere.

**Design Description:**
having this as a `rootchain_contract` field on `/status.get` endpoint.

**Comment by pnowosie** [comment]
This is align with the idea described in #526 (which after EIP-712 is ready to think about seriously)
I just wanted to point out that client to sign need the whole EIP `domain` data, consider to extend the endpoint of

```javascript
domainData: {
    name: "OMG Network",
    version: "1",
    verifyingContract: "0x44de0ec539b8c4a4b530c78620fe8320167f2f74",
    salt: "0xfad5c7f626d80f9256ef01929f3beb96e058b8b4b0e3fe52d84f054c0e2a7a83"
}
```

**Comment by pnowosie** [comment]
Watcher reports contract address in [status.get endpoint](https://developer.omisego.co/elixir-omg/docs-ui/?url=master/security_critical_api_specs.yaml#/Status/status_get)

**Comment by pdobacz** [comment]
Ha! so fixed in #708. I didn't even know I solved an issue then :D
